common/planners/src/plan_display.rs

```diff
@@ -62,7 +62,7 @@ impl PlanNode {
                                     f,
                                     "{:?}:{:?}",
                                     plan.exprs[i],
-                                    plan.schema().fields()[i].data_type()
+                                    plan.exprs[i].to_data_type(&plan.input.schema()).unwrap()
```

Comment:
`unwrap` can cause panic. use `?` operator

Reply:
It's for display, may need ErrorCodes to fmt error